src/data/oppfolgingsoppgave/useDeleteAndCreateOppfolgingsoppgave.ts

```diff
@@ -0,0 +1,33 @@
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { deleteOppfolgingsoppgave } from "@/data/oppfolgingsoppgave/useRemoveOppfolgingsoppgave";
+import { postOppfolgingsoppgave } from "@/data/oppfolgingsoppgave/useCreateOppfolgingsoppgave";
+import { OppfolgingsoppgaveRequestDTO } from "@/data/oppfolgingsoppgave/types";
+import { aktivOppfolgingsoppgaveQueryKeys } from "@/data/oppfolgingsoppgave/useAktivOppfolgingsoppgave";
+import { oppfolgingsoppgaverQueryKeys } from "@/data/oppfolgingsoppgave/useOppfolgingsoppgaver";
+
+export const useDeleteAndCreateOppfolgingsoppgave = () => {
```

Comment:
Jeg tenker på dette som en detalj som er viktig internt i domenet mtp. på hvordan vi lagrer det, og ikke er en detalj som en klient skal måtte forholde seg til. Om vi håndterer dette utenfor appen som eier domenet, så legger vi opp til at dette er en detalj man må huske på ved å spesifikt se på hvordan klienten interagerer med appen når årsaken endres.
Ser for meg at dette kunne blitt løst ved å definere denne oppførselen i `ishuskelapp` spesielt for når en bruker ønsker å editere årsaken til oppfølging.
Hva tenker du? 😊

Reply:
Kan være det blir ryddigere og enklere, ja. Tenker du da at vi bare tilpasser `POST`-endepunktet vi har for editering i dag til å behandle endret oppfolgingsgrunn spesielt? Slik at frontend bare trenger å tillatte endring av feltet.

Reply:
Det var slik jeg så det for meg. Kan godt være det er problemer ved å gjøre akkurat dette også altså🤔 Virker bare som en oppførsel vi ønsker å dokumentere i `ishuskelapp`.

Reply:
Kan se litt på hvor lett det er å få til i backend. For så vidt enig i at det hadde vært fint å unngå kompleksiteten med å gjøre to kall her i frontend også, mtp feilhåndtering osv.